lite/benchmarks/benchmark_objdet.py

```diff
@@ -8,7 +8,7 @@
 
 import requests
 from tqdm import tqdm
-from valor_lite.detection import DataLoader, MetricType
+from valor_lite.cv.detection import DataLoader, MetricType
```

Comment:
the "CV" bucket doesn't feel MECE: if the user does `pip install valor-<name>[cv]` (from your example yesterday), they'd expect classification to be installed too. why not leave classification, detection, and segmentation as-is (i.e., not grouping them under CV), but group the LLM metrics separately under NLP?

Reply:
Polling for feedback on flattening the import structure! 

As for the custom installs, I feel thats something we can deal with at a later point.